### PR TITLE
Added modal confirmation before actually deleting ads and wishlist items

### LINKED ISSUE #195: [User Story 17] Prompt Before Deleting ad
### Prompt Before Deleting ad

| Risk | Priority | Story Points |
| :---: | :---: | :---: | 
| High | Low | **2** |

As a a User when I am deleting an ad I should be prompted to confirm the deletion before it is officially deleted from the database.

- [ ] 17.1 Create popup that is triggered when delete is clicked #??? (**2 Hours**) 

#### Total: **2 Hours**

### PR DESCRIPTION
**Pull request by BrandonBalala**
Resolves #195

### CODE REVIEW

File: public/js/script.js [hunk 1]
```diff
@@ -9,12 +9,29 @@ function updateQueryStringParameter(uri, key, value) {
...
 }
 
+/*Adding sort by parameter to the URL*/
 $('.sortBy').on('click',function(){
 	var URL = $(location).attr("href");
 	var newURL = updateQueryStringParameter(URL, 'sortBy', this.id);
 	window.location.href = newURL;
 });
 
+/*Deals with modal confirmation prompt for deleting your own ad*/
```
Comment (kevcamel) [inline public/js/script.js:19]: good job with the commenting

File: resources/views/yourstore.blade.php [hunk 2]
```diff
@@ -52,11 +52,9 @@
 		                </form>
 		                @endif
 		                @if(Auth::check() && $user->id == Auth::user()->id)
-						<form  action="/ad/{{ $product->id }}" method="POST">
-			                {{ csrf_field() }}
-			                <input type="hidden" name="_method" value="DELETE">
-			                <button id="deleteWishlistBtn" type="submit" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-		                </form>
+		                <button type="button" class="close" aria-label="Close" data-toggle="modal" data-target="#deleteAdModal" data-productid="{{ $product->id }}">
+		                	<span aria-hidden="true">&times;</span>
```
Comment (kevcamel) [inline resources/views/yourstore.blade.php:56]: I have been getting rid of the  ARIA labels (aria-hidden, aria-visible etc...) its for reader only, I find it clutters up he code too mich. Any thoughts? @BrandonBalala

**Comment by kevcamel** [comment]
screen shot 2017-11-08 at 7 02 46 pm
screen shot 2017-11-08 at 7 02 26 pm
Looks great 👍

File: resources/views/wishList.blade.php [hunk 3]
```diff
@@ -5,38 +5,51 @@
...
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+    	<span>
+			<form  id="deleteWishlistItemForm" action="" method="POST">
+	            {{ csrf_field() }}
+	            <input type="hidden" name="_method" value="DELETE">
+	            <button id="deleteWishlistBtn" type="submit" class="btn btn-danger">Delete</button>
+	        </form>
+		</span>
```
Comment (kevcamel) [inline resources/views/wishList.blade.php]: is it just github that messed up the indentation here?

**Comment by Swallow666** [comment]
Removing ad in wishlist still doesn't work on my local and it jumps to wishlist/7

**Comment by BrandonBalala** [comment]
So the problem was that instead of passing the id of the wishlist item that I wanted to delete, I was passing the product id. @Swallow666 try it out and see that it works now?

**Comment by Swallow666** [comment]
I tested, it works well now.

**Review by neqqash — COMMENTED** [review]
Coding style is enforced, everything else seems good too. Simple to follow

**Pull request merged**